css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+body {
+  background-color: rgb(25, 150, 218);
+}
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.column-full {
+  text-align: center;
+}
+
+.column-third {
+  width: 33%;
+  flex-wrap: wrap;
```

Comment:
This can be removed, this class is not a flexbox since it doesn't have `display: flex`, so this line does nothing.